tools/pcxtool/main.c

```diff
@@ -91,14 +92,6 @@ static void show_pcx(pcx_file *pcx) {
     }
 }
 
-static int file_exists(char const *filename) {
```

Comment:
With the removal of file_exists, could you not remove the `<unistd.h>` and `<io.h>` includes?
the if-windows <errno.h> has always been redundant, as well.

Reply:
Yeah, was able to remove some includes related to this here and in pathmanager.